tests/config_tests.rs

```diff
@@ -0,0 +1,418 @@
+#[cfg(test)]
```

Comment:
@pyakushin where proper place for test in this case?

Reply:
In the `config` mod, tests in project root are integration for crate.
[Rust Book Tests](https://doc.rust-lang.org/book/ch11-03-test-organization.html#the-tests-module-and-cfgtest)